<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanetWeightDialog</class>
 <widget class="QDialog" name="PlanetWeightDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>275</width>
    <height>450</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>275</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Planetary Weight Calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Enter Weight:</string>
       </property>
       <property name="buddy">
        <cstring>weightLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="weightLineEdit"/>
     </item>
     <item>
      <widget class="QComboBox" name="unitsComboBox">
       <item>
        <property name="text">
         <string>lbs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>kg</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="calcWeightButton">
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="aboutButton">
       <property name="text">
        <string>About</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sun</string>
       </property>
       <property name="buddy">
        <cstring>sunLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="sunLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Moon</string>
       </property>
       <property name="buddy">
        <cstring>moonLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="moonLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Mercury</string>
       </property>
       <property name="buddy">
        <cstring>mercuryLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="mercuryLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Venus</string>
       </property>
       <property name="buddy">
        <cstring>venusLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="venusLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Mars</string>
       </property>
       <property name="buddy">
        <cstring>marsLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="marsLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="toolTip">
        <string>Dwarf Planet (IAU 2006)</string>
       </property>
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;Ceres&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="buddy">
        <cstring>ceresLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="ceresLineEdit">
       <property name="toolTip">
        <string extracomment="Dwarf Planet (IAU 2006)">Dwarf Planet (IAU 2006)</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Jupiter</string>
       </property>
       <property name="buddy">
        <cstring>jupiterLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="jupiterLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Saturn</string>
       </property>
       <property name="buddy">
        <cstring>saturnLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="saturnLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Uranus</string>
       </property>
       <property name="buddy">
        <cstring>uranusLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="uranusLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Neptune</string>
       </property>
       <property name="buddy">
        <cstring>neptuneLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="neptuneLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_10">
       <property name="toolTip">
        <string>Dwarf Planet (IAU 2006)</string>
       </property>
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-style:italic;&quot;&gt;Pluto&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="buddy">
        <cstring>plutoLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="plutoLineEdit">
       <property name="toolTip">
        <string>Dwarf Planet (IAU 2006)</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>weightLineEdit</tabstop>
  <tabstop>calcWeightButton</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>aboutButton</tabstop>
  <tabstop>quitButton</tabstop>
  <tabstop>sunLineEdit</tabstop>
  <tabstop>moonLineEdit</tabstop>
  <tabstop>mercuryLineEdit</tabstop>
  <tabstop>venusLineEdit</tabstop>
  <tabstop>marsLineEdit</tabstop>
  <tabstop>ceresLineEdit</tabstop>
  <tabstop>jupiterLineEdit</tabstop>
  <tabstop>saturnLineEdit</tabstop>
  <tabstop>uranusLineEdit</tabstop>
  <tabstop>neptuneLineEdit</tabstop>
  <tabstop>plutoLineEdit</tabstop>
  <tabstop>unitsComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>PlanetWeightDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>weightLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>sunLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>moonLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>mercuryLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>venusLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>marsLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>ceresLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>jupiterLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>saturnLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>uranusLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>neptuneLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>plutoLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
